### ISSUE TITLE
parser: fill states and intents in appdef for cmds and queries

### ISSUE THREAD

**Issue by michael-say** [voedger/voedger#1670]

**Comment by michael-say** [comment]
@mail2nnv `IExtensionBuilder` doens't implement `Intents()` and `States()`, you can fix it [in the branch](https://github.com/michael-say/voedger/tree/1670-parser-build-state-and-intens-for-commands-appdef)

**Comment by mail2nnv** [comment]
> @mail2nnv `IExtensionBuilder` doens't implement `Intents()` and `States()`, you can fix it [in the branch](https://github.com/michael-say/voedger/tree/1670-parser-build-state-and-intens-for-commands-appdef)

Can not fix in you branch
<details><summary>Details</summary>
<p>

```
> git push michael-say 1670-parser-build-state-and-intens-for-commands-appdef:1670-parser-build-state-and-intens-for-commands-appdef
git: 'credential-manager-core' is not a git command. See 'git --help'.
To https://github.com/michael-say/voedger
 ! [remote rejected] 1670-parser-build-state-and-intens-for-commands-appdef -> 1670-parser-build-state-and-intens-for-commands-appdef (permission denied)
error: failed to push some refs to 'https://github.com/michael-say/voedger'
```

</p>
</details> 

I made fix at my branch, see issue #1671
Please run ```git pull upstream main``` to receive my changes from voedger/voedger main
